### ISSUE TITLE
Find My Device Network Support

### ISSUE THREAD

**Issue by ojoeth** [GrapheneOS/os-issue-tracker#4079]
I'm having trouble getting my new Pebblebee Clip to work with my Pixel 8 Pro running GrapheneOS. Here's what I've tried:

1. Direct pairing fails (see screenshot)
2. Installed Find My Device app and granted all permissions
3. Attempted to enable "find my device network" settings:
   - "With network in high-traffic areas only"
   - "With network in all areas"
   Both options fail (screenshot below)

I found a related thread (https://discuss.grapheneos.org/d/13595-find-my-device-and-tracker) suggesting a workaround:
- Pair the Pebblebee with a secondary phone running stock Android
- This allows viewing the tracker's last known location
- Downside: GrapheneOS phone can't scan independently, and secondary phone must always be on

Is there a way to get the Pebblebee Clip working directly with GrapheneOS? Or are there any plans to improve compatibility with these types of trackers?

Any help or insights would be appreciated. Thanks!

![JPEG_20240930_202453_3499995724160890292](https://github.com/user-attachments/assets/b4a1a7d4-6534-4f10-9f2a-217219b8c8a1)
![JPEG_20240930_202443_7889034852084084815](https://github.com/user-attachments/assets/79503e0b-092d-4b34-bc45-d3b5f5b9f3a8)

**Comment by w00ter** [comment]
Exactly same case with Moto Tags, would very much love for this to be resolved. None of the elevevated play permissions work. 

@thestinger is this something the team is willing to reconsider?

**Comment by Qxji** [comment]
I would love to see it! Especially as the second phone work-around sucks.

**Comment by jstride** [comment]
Yup, just come across this one too and I have a few pebblebee I'd like to use.

**Comment by w00ter** [comment]
Hey everyone,

After last GOS Android 15 update and/or GMS related updates by the GOS team, I can confirm this now works on my side.

Moto Tags in Find My Device can now be located and tracked. 

I also noticed a toggle 'allow bluetooth scanning' alongside WiFi scanning, that may have already been there, so make sure to enable. 

Thank you GrapheneOS !

**Comment by Whatnoww** [comment]
Pairing still does not work for me. Yet locating via the Find My app has just started working for me too.

**Comment by w00ter** [comment]
> Pairing still does not work for me. Yet locating via the Find My app has just started working for me too.

Same here unfortunately, I cannot pair a new tag either. At least we're one step closer!

**Comment by baptiste-mnh** [comment]
I can see the devices that I added from another Android device but I cannot connect to them or make them ring neither.

**Comment by edent** [comment]
How are you able to see shared devices? I've added the PebbleBee to a normal Android phone - but when I try to share it with my GrapheneOS device, I get this error:

![Device sharing is not available on this device.](https://github.com/user-attachments/assets/061ee193-8a68-4ab9-8dff-a26c5966d98b)

Clicking "Learn More" doesn't provide any useful information.

I'm running the latest GOS on a Pixel 8 Pro. I think I've given all the correct permissions to Find My Device and to Google Play.  Does anyone know how I can fix the "Device sharing is not available on this device" problem?

**Comment by jelee1988** [comment]
> Google Play

It would be nice if it is considered.

**Comment by gh69** [comment]
> How are you able to see shared devices? I've added the PebbleBee to a normal Android phone - but when I try to share it with my GrapheneOS device, I get this error:
> 
> ![Device sharing is not available on this device.](https://github.com/user-attachments/assets/061ee193-8a68-4ab9-8dff-a26c5966d98b)
> 
> Clicking "Learn More" doesn't provide any useful information.
> 
> I'm running the latest GOS on a Pixel 8 Pro. I think I've given all the correct permissions to Find My Device and to Google Play. Does anyone know how I can fix the "Device sharing is not available on this device" problem?

I don't believe _**sharing**_ works...ie sharing a tracker from one Google Play account with a different Google Play account...but if you use a single Google Play account on a stock android phone to add your trackers via FMD and then logon to that same Google Play account on your GOS phone, you should be able to see those trackers, locate them, and ring them.

**Comment by malloy139** [comment]
You can add a tracker to your Google account with a different phone without GOS

**Comment by Titaniumtown** [comment]
I was able to connect to it via Google settings, but i wasnt able to add it to findhub: 

![Image](https://github.com/user-attachments/assets/d5719d65-6efc-41da-b5e8-daff854772cc)

**Comment by w00ter** [comment]
Just tried pairing with my Moto Tag again, no luck. It is recognized and presents a pairing code, then fails to pair. 

:-(

**Comment by cuevasclemente** [comment]
Can confirm as well -- I can find/make sound from a moto tag I paired with my phone before I installed GOS on it, but I cannot pair a new one on my GOS phone. Happy to help test any attempts related to this!

**Comment by jongrall** [comment]
Bummer, I just learned that these kinds of trackers aren't yet supported after I bought some pricey ones from Chipolo. Hoping this feature makes it into a future update soon.

**Comment by edent** [comment]
I'm happy to buy the GrapheneOS developers a PebbleBee clip if it will help get this feature added :-)

**Comment by malloy139** [comment]
Since one of the latest updates, UWB now works. It displays the distance accurately to within 0.1 meters, but the direction arrow does not work yet

**Comment by kgu947** [comment]
Just a +1 - would love to get this working. Seems I'm not alone.

**Comment by jongrall** [comment]
> Since one of the latest updates, UWB now works. It displays the distance accurately to within 0.1 meters, but the direction arrow does not work yet

Even if UWB now partially works – I am unable to verify that it does – GrapheneOS does not seem to play nicely with Google's Find Hub if you install it. I cannot add any UWB devices using Google Find Hub on GrapheneOS. I suppose if you had a standard Android device to complete the device adding process, then locating devices that you add that way might work, but I don't have another Android device to try this with, and so I am unable to proceed any further.

**Comment by NoSadBeHappy** [comment]
> I suppose if you had a standard Android device to complete the device adding process, then locating devices that you add that way might work, but I don't have another Android device to try this with, and so I am unable to proceed any further.

This is correct, I was able to verify that it works with another device being used to set it up, and have been using it for a few months without issue.

**Comment by otosan0** [comment]
> but if you use a single Google Play account on a stock android phone to add your trackers via FMD and then logon to that same Google Play account on your GOS phone, you should be able to see those trackers, locate them, and ring them.

After adding a tracker through another android device I can see the tracker in the Find My app but I am not able to see its location. It says "Location unavailable" on GOS and a red box with "Sync recent location - You may need to sign in or enter your screen lock to view the location" (FTR: I am signed in and already entered my screen code to open the app).

It is possible to "Play a sound" but only when the other android phone is connected to the tracker.

How do you guys get the location visible on the GOS phone in Find My app? I can only see it on my other android device.

**Comment by vn971** [comment]
I'm now trying with the "UGREEN FineTrack G Tracker for Android" on GrapheneOS Pixel 8. Same as reported for other devices, pairing fails. From the pull request linked here 2 weeks ago, it seems that no Find Hub tracker pairing will work until GOS implements a work-around.

**Comment by nouritsu** [comment]
Trying with the FnR 6ATAG72R and FnR 6ACTAG7R from "Fresh 'n Rebel" (EU brand)
- the tag pops up as a fast pair request
- connecting..... then "connected" for a brief moment
- (screenshot)

![Screenshot_20260828-235108.png](https://github.com/user-attachments/assets/8f8142f1-21d2-4de5-9557-23939500db66)

This may be a good tag for developers to test around with as they can be purchased in Action for just under 7€ (Germany).

**Comment by DevTheFool** [comment]
> Trying with the FnR 6ATAG72R and FnR 6ACTAG7R from "Fresh 'n Rebel" (EU brand)
> 
> * the tag pops up as a fast pair request
> * connecting..... then "connected" for a brief moment
> * (screenshot)
> 
> ![Screenshot_20260828-235108.png](https://github.com/user-attachments/assets/8f8142f1-21d2-4de5-9557-23939500db66)
> 
> This may be a good tag for developers to test around with as they can be purchased in Action for just under 7€ (Germany).

I have the same tag, I don't even get a pop up to connect, I have no idea how to make it work.
